src/qt/bitcoinstrings.cpp

```diff
@@ -9,111 +9,111 @@
 #define UNUSED
 #endif
 static const char UNUSED *bitcoin_strings[] = {
-QT_TRANSLATE_NOOP("bitcoin-core", ""
+QT_TRANSLATE_NOOP("bitcoin-classic", ""
 "(1 = keep tx meta data e.g. account owner and payment request information, 2 "
 "= drop tx meta data)"),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
+QT_TRANSLATE_NOOP("bitcoin-classic", ""
 "Allow JSON-RPC connections from specified source. Valid for <ip> are a "
 "single IP (e.g. 1.2.3.4), a network/netmask (e.g. 1.2.3.4/255.255.255.0) or "
 "a network/CIDR (e.g. 1.2.3.4/24). This option can be specified multiple times"),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
+QT_TRANSLATE_NOOP("bitcoin-classic", ""
 "An error occurred while setting up the RPC address %s port %u for listening: "
 "%s"),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
+QT_TRANSLATE_NOOP("bitcoin-classic", ""
 "Bind to given address and always listen on it. Use [host]:port notation for "
 "IPv6"),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
+QT_TRANSLATE_NOOP("bitcoin-classic", ""
 "Bind to given address and whitelist peers connecting to it. Use [host]:port "
 "notation for IPv6"),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
+QT_TRANSLATE_NOOP("bitcoin-classic", ""
 "Bind to given address to listen for JSON-RPC connections. Use [host]:port "
 "notation for IPv6. This option can be specified multiple times (default: "
 "bind to all interfaces)"),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
-"Cannot obtain a lock on data directory %s. Bitcoin Core is probably already "
+QT_TRANSLATE_NOOP("bitcoin-classic", ""
+"Cannot obtain a lock on data directory %s. Bitcoin Classic is probably already "
```

Comment:
Does Classic use a different data directory lock or would it also be locked if Bitcoin Core is already running?
Should it say "A Bitcoin client is probably already running."?